specs/DesignDiscussions/InputLabelsOptionLabel.md

```diff
@@ -0,0 +1,440 @@
+# Input/Label Associations
+
+This document covers design considerations related to issue [#203](https://github.com/microsoft/AdaptiveCards/issues/203).
+
+Consider the following card:
+```json
+{
+	"type": "AdaptiveCard",
+	"version": "1.0",
+	"body": [
+		{
+			"type": "TextBlock",
+			"text": "Please enter your name:"
+		},
+		{
+			"type": "Input.Text",
+			"id": "nameInput",
+ 			"placeholder": "Last, First"
+		}
+	],
+	"actions": [
+		{
+			"type": "Action.Submit",
+			"title": "OK"
+		}
+	]
+}
+```
+
+![img](assets/InputLabels/InputCard.png)
+
+In that card, the label for the input is “Please enter your name:”, however the card renderer has no way to determine that this is the case. We’ve run into a few cases where this is an issue:
+ - Accessibility: The input will be read by screen readers as “Last, First”, rather than “Please enter your name” making it more difficult for screen reader users to use the card. 
+
+ - Validation indicators: As part of input validation, we are allowing card authors to mark inputs as optional or required. One way to indicate that visually would be to put an indicator next to the input’s label. This requires knowing which text is the label.
+
+ - Interaction model: Many forms move focus to the input when the label is clicked/tapped on. We can’t support that behavior without knowing which label goes with which input.
+
+ - Responsiveness: One direction for the future of cards involves re-flowing the layout of a card based on the size of the display. Knowing that a particular label goes with a particular input may help us ensure that semantically related elements stay together on reflow.
+
+## Basic Label Support in Adaptive Cards
+
+The basic addition that we are going to make to the schema is to add a label property to all input types. This approach provides a simple and concise way to label inputs, that hopefully encourages card authors to use it and receive the benefits of improvements to the issues called out above.
+
+``` json
+{
+	"type": "AdaptiveCard",
+	"version": "1.0",
+	"body": [
+		{
+			"type": "Input.Text",
+			"id": "nameInput",
+			"label": "Please enter your name:",
+			"placeholder": "Last, First"
+		}
+	],
+	"actions": [
+		{
+			"type": "Action.Submit",
+			"title": "OK"
+		}
+	]
+}
+```
+
+## Formatting
```

Comment:
- How important of a feature is card author formatting support and can this be included in InputValidation part 2? @paulcam206 / @siduppal ; Can we have simple strings here as values for starters? 
- What's the min-bar here wrt formatting as far as Accessibility is concerned?

Reply:
I personally think that card author formatting is less important than host formatting